pkg/dashboard/ui/package.json

```diff
@@ -48,7 +48,7 @@
     "i18next-chained-backend": "^3.0.2",
     "i18next-localstorage-backend": "^3.1.3",
     "i18next-xhr-backend": "^3.2.2",
-    "iguazio.dashboard-controls": "^1.0.13",
+    "iguazio.dashboard-controls": "^1.1.1",
```

Comment:
```suggestion
    "iguazio.dashboard-controls": "1.1.1",
```

According to [this](https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json):

> `^version` : “Compatible with version”, will automatically update you to all future minor/patch versions that are backwards-compatible, without incrementing the major version. ^1.2.3 will use releases from 1.2.3 to < 2.0.0.

Using `^` is bad here, because the backwards compatibility is not kept between minor versions in Dashboard-controls.
For instance, a change was made in `1.1.0` that broke the Nuclio CI, before it was bumped here.
If Backwards compatibility is kept between patch versions, we can use `~`, however if not - we need to be strict and not let NPM resolve the latest images.

FYI @ilan7empest